nwbinspector/nwbinspector.py

```diff
@@ -63,12 +65,20 @@ def inspect_all_cli(
     ignore: Optional[str] = None,
     select: Optional[str] = None,
     threshold: str = "BEST_PRACTICE_SUGGESTION",
+    config_path: Optional[str] = None,
     json_file_path: str = None,
 ):
     """Primary CLI usage."""
+    if config_path is not None:
+        with open(file=config_path, mode="r") as stream:
+            config = yaml.load(stream, yaml.Loader)
```

Comment:
Should this perform

```python
with open(file=Path(__file__).parent / "config.schema.json", mode="r") as fp:
    schema = json.load(fp=fp)
jsonschema.validate(config, schema)
```

do you think?

Reply:
```python
file=Path(__file__).parent / "config.schema.json"
```

I don't like hard-coding this. We should allow the user to provide their own config file from anywhere they want to store it.

Validating seems like a good idea though.

Reply:
I see. Sure.